### ISSUE TITLE
demo-vectortiles for MVT giving 401 unauthorised errors

### ISSUE THREAD

**Issue by mvl22** [Leaflet/Leaflet.VectorGrid#126]
http://leaflet.github.io/Leaflet.VectorGrid/demo-vectortiles.html

in the MVT mode is not working, as the tiles are giving a 401 response.

**Comment by perliedman** [comment]
@IvanSanchez is the mapbox tiles' token from your account? Have we exceeded the quota? Possibly someone stole the token and used in their app (that has happened to me a couple of times)?

**Comment by IvanSanchez** [comment]
Yeah, the failure is only for the mapbox vector tiles, and yes, I exceeded the quota some time ago.

This should be a simple matter of changing the mapbox token in the demo.
